pkg/management/http.go

```diff
@@ -79,11 +79,11 @@ func decodeManagementRequest(ctx context.Context, req *http.Request) (interface{
 		prmAction:       constants.RegExpName,
 	}
 
-	var queryParams = map[string]string{
-		prmQryEmail:         constants.RegExpEmail,
+	queryParams := map[string]string{
+		prmQryEmail:         constants.RegExpSearch,
 		prmQryFirstName:     constants.RegExpSearch,
-		prmQryLastName:      constants.RegExpLastNameSearch,
```

Comment:
So RegExpLastNameSearch and RegExpUsername are still in use?

Reply:
Apparently, RegExpLastNameSearch can be safely removed but RegExpUsername is used [here](https://github.com/cloudtrust/keycloak-bridge/blob/master/pkg/kyc/http.go).